src/ComponentCard.jsx

```diff
@@ -160,7 +176,7 @@ return (
       </CardTag>
     )}
 
-    <CardBody>
+    <CardBody style={{padding : props.metadata?.star_count>=0 && props.metadata.fork_count>=0 && '16px 16px 4px 16px'}}>
```

Comment:
Let's avoid using inline styles. Can we move this to the styled component declaration?

Reply:
Yep, Since we dont have the historical backfill for the indexer yet - I have a condition for styling in case the component does not have stars and forks. After the backfill all components will have a star and fork count (even if its 0) so we can move it to a stylized component.